x-pack/plugins/monitoring/public/components/cluster/overview/index.js

```diff
@@ -12,6 +12,7 @@ import { AlertsPanel } from './alerts_panel';
 import { BeatsPanel } from './beats_panel';
 
 import { EuiPage, EuiPageBody } from '@elastic/eui';
+import { ApmPanel } from './apm_panel';
```

Comment:
Nit: Does KB have any rules on how imports should be sorted? Judging on the list above probably not :-)